src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -89,6 +91,7 @@ public void setupSchema(List<ColumnMetadata> columns) {
     if (!clientBufferParameters.isEnableIcebergStreaming()) {
       metadata.put("sfVer", "1,1");
     }
+    metadata.put(Constants.SDK_VERSION_KEY, RequestBuilder.DEFAULT_VERSION);
```

Comment:
nice, Alec and I had discussed doing precisely this just yesterday!

We're going to have multiple SDK languages in a few months, should we have a slightly more verbose key-value pair here, such as:
userAgent=RequestBuilder.USER_AGENT ?

Reply:
User-Agent is a very HTTP-ish thing, we could name the key createdBy instead.

Reply:
This current key-value is what XP expects. Let's iterate on it later?